x/mongo/driver/operation.go

```diff
@@ -58,6 +58,11 @@ type RetryablePoolError interface {
 	Retryable() bool
 }
 
+// LabelledError is an error that can have error labels added to it.
```

Comment:
```suggestion
// LabeledError is an error that can have error labels added to it.
```
[nit] let's use the [American spelling](https://sapling.ai/usage/labeled-vs-labelled) for consistency with [existing code](https://github.com/mongodb/mongo-go-driver/blob/master/mongo/errors.go#L199).